README.md

```diff
@@ -7,7 +7,9 @@
 
 # gomplate
 
-A [Go template](https://golang.org/pkg/text/template/)-based alternative to [`envsubst`](https://www.gnu.org/software/gettext/manual/html_node/envsubst-Invocation.html).
+A [Go template](https://golang.org/pkg/text/template/)-based CLI tool. `gomplate` can be used as an alternative to
+[`envsubst`](https://www.gnu.org/software/gettext/manual/html_node/envsubst-Invocation.html) but also supports
+additional template datasources such as JSON, YAML, AWS metadata, and vault secrets.
```

Comment:
I think `vault` should be `Vault` (or `Hashicorp Vault`), and should link to `https://vaultproject.io`. Lots of potential users aren't familiar with it, and would probably benefit greatly by it 😉